Make it clear that customer should not use POA and AutomaticOSUpgrades (#79398)
Azure/azure-content: articles/service-fabric/how-to-patch-cluster-nodes-windows.md

With Bronze durability, automatic OS image upgrade isn't available. While [Patch Orchestration Application](service-fabric-patch-orchestration-application.md) (intended only for non-Azure hosted clusters) is *not recommended* for Silver or greater durability levels, it is your only option to automate Windows updates with respect to Service Fabric upgrade domains.

> When using automatic OS image upgrades you should deprecate the use of the Patch Orchestration Application in the case you were using it before.

Reviewer: ```suggestion
If you want to switch from Patch Orchestration Application to automatic OS image upgrade, you must first deprecate the use of Patch Orchestration Application.
```
